src/vm/eetwain.cpp

```diff
@@ -562,7 +562,8 @@ FrameType   GetHandlerFrameInfo(hdrInfo   * info,
     // The slots grow towards lower address on the stack and is terminted by a NULL entry.  
     // Since each subsequent slot contains the SP of a more nested EH clause, the contents of the slots are
     // expected to be in decreasing order.
-    size_t lvl;
+    size_t lvl = 0;
```

Comment:
@parjong looking at the code that calls this function, I wonder if it should be called at all for x86 with WIN64EXCEPTIONS. Most of the callers do nothing if the _TARGET_X86_ is not defined and it seems like these ifdefs should be augmented for _TARGET_X86_ && !WIN64EXCEPTIONS.
I am not 100% sure though.
@jkotas do you know if I am right here or will I need to look through the code to get full understanding?